Sources/MusicPlayer/Players/SystemMedia.swift

```diff
@@ -164,4 +164,93 @@ private extension Notification.Name {
     static let mediaRemoteNowPlayingApplicationPlaybackStateDidChange = Notification.Name("kMRMediaRemoteNowPlayingApplicationPlaybackStateDidChangeNotification")
 }
 
+#elseif os(Linux)
+
+import playerctl
+
+extension MusicPlayers {
+    
+    public final class FakeSystemMedia: Agent {
+        
+        private let manager: UnsafeMutablePointer<PlayerctlPlayerManager>
+        
+        private var players: [MusicPlayers.MPRIS] = []
+        private var signals: [gulong] = []
+        
+        public init?(_: Void = ()) {
```

Comment:
why void parameter? I think `init?()` is sufficient.

Reply:
In Swift, a failable initializer cannot override a non-failable initializer, add `Void` parameter to change the function signature.

Reply:
Using NowPlaying need no void argument because of different signature.